forge-gui/res/adventure/Shandalar/custom_cards/slimefoots_slimy_staff.txt

```diff
@@ -0,0 +1,8 @@
+Name:Slimefoot's Slimy Staff
+ManaCost:no cost
+Colors:black,green
+Types:Artifact Equipment
+A:AB$ MakeCard | Cost$ G PayShards<2> | Name$ Bayou | ActivationZone$ Command | Zone$ Hand | IsPresent$ Land.YouOwn | PresentZone$ Hand | PresentCompare$ EQ0 | GameActivationLimit$ 1 | SpellDescription$ Conjure a card named Bayou into your hand. Activate only if there are no land cards in your hand and only once each game. 
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Command | Execute$ TrigPutCounter | TriggerDescription$ At the beginning of your end step, put a spore counter on each fungus you control.
```

Comment:
I've got intentions of reviewing all items with in-match effects to ensure that everything that impacts a match should have a Mana Shard cost to activate - we can tweak the economy suitably later to ensure the players are receiving enough shards to pull the trigger on using these while not being able to hold it down. So while non-binding, I would prefer this to be an activated ability with a shard cost to create this perpetual effect, and to be mutually exclusive with the Bayou conjuring. Keep it as is for now if you like though and we can hammer that out later.